src/components/app/index.tsx

```diff
@@ -233,7 +249,7 @@ export default class App extends Component<Props, State> {
       loadedCounter: loadingCounter,
     };
 
-    this.setState({ images, error: '' });
+    this.setState({ images });
```

Comment:
Why not clearing the previous error?

Reply:
I think this got merged into master in the meantime... I don’t remember removing it myself

Reply:
It's still there on master